qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -73,19 +72,40 @@ def format(self) -> str:
         Returns:
              The job queue information report.
         """
-        job_status = api_status_to_job_status(ApiJobStatus(self._status))
-        estimated_start_time = arrow.get(self.estimated_start_time).humanize()
-        estimated_completion_time = arrow.get(self.estimated_complete_time).humanize()
+        _default_attr_value = 'unknown'
+        attributes_map = self._get_attributes_map(_default_attr_value)
+
+        status = attributes_map['_status']
+        if status != _default_attr_value:
+            attributes_map['_status'] = api_status_to_job_status(ApiJobStatus(status)).value
+
+        est_start_time = attributes_map['estimated_start_time']
+        if est_start_time != _default_attr_value:
+            attributes_map['estimated_start_time'] = arrow.get(est_start_time).humanize()
+
+        est_complete_time = attributes_map['estimated_complete_time']
+        if est_complete_time != _default_attr_value:
+            attributes_map['estimated_complete_time'] = arrow.get(est_complete_time).humanize()
 
         queue_info = [
-            "Job {} queue information:".format(self.job_id),
-            "    queue position: {}".format(self.position),
-            "    status: {}".format(job_status.value),
-            "    estimated start time: {}".format(estimated_start_time),
-            "    estimated completion time: {}".format(estimated_completion_time),
-            "    hub priority: {}".format(self.hub_priority),
-            "    group priority: {}".format(self.group_priority),
-            "    project priority: {}".format(self.project_priority)
+            "Job {} queue information:".format(attributes_map['job_id']),
+            "    queue position: {}".format(attributes_map['position']),
+            "    status: {}".format(attributes_map['_status']),
+            "    estimated start time: {}".format(attributes_map['estimated_start_time']),
+            "    estimated completion time: {}".format(attributes_map['estimated_complete_time']),
+            "    hub priority: {}".format(attributes_map['hub_priority']),
+            "    group priority: {}".format(attributes_map['group_priority']),
+            "    project priority: {}".format(attributes_map['project_priority'])
```

Comment:
I think it's better to avoid using strings as attribute names, since it is prone to typos not caught by the IDE and more easily missed when the attribute is renamed. You can instead write a small function that takes `foo` and returns `foo` if not `None` or "unknown".